src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -387,7 +387,10 @@ class HttpGoogleServicesDAO(
       case Right(member) => new Member().setEmail(member.groupEmail.value).setRole(groupMemberRole)
     }
     val inserter = getGroupDirectory.members.insert(toGoogleGroupName(groupRef.groupName), memberEmail)
-    retry(when500)(() => Future { blocking { inserter.execute } })
```

Comment:
changes in this file are important because, in dev, we have lots of rawlses that fiddle with groups that may already have members we are trying to add or not have members we are trying to delete. It will also come in handy when blowing away an existing database.